### PR TITLE
[REGEDIT] Update Romanian (ro-RO) translation

### PR DESCRIPTION
**Pull request by AndreiMiloiu**
Attendum to https://github.com/reactos/reactos/pull/6517

This files was not translated in Romanian language in any Win version (neither XP/2k3+, nor in the previous ones). I tried to translate this file in the same way I did in my previous PRs with this attendum.

### CODE REVIEW

**Comment by AndreiMiloiu** [comment]
Cc @reactos/lang-romanian

File: base/applications/regedit/lang/ro-RO.rc [hunk 1]
```diff
@@ -83,57 +83,57 @@ BEGIN
...
         MENUITEM "&Bara de stare", ID_VIEW_STATUSBAR
         MENUITEM SEPARATOR
-        MENUITEM "&Reglează separator", ID_VIEW_SPLIT
+        MENUITEM "&Reglare a separatorului", ID_VIEW_SPLIT
         MENUITEM SEPARATOR
-        MENUITEM "Împ&rospătează\tF5", ID_VIEW_REFRESH
+        MENUITEM "Î&mprospătare\tF5", ID_VIEW_REFRESH
     END
-    POPUP "Fa&vorite"
+    POPUP "&Favorite"
```
Comment (JoachimHenze) [inline base/applications/regedit/lang/ro-RO.rc:129]: Change requested: The &Favorite would collide with "&Fișier". Change is mandatory for that.

File: base/applications/regedit/lang/ro-RO.rc [hunk 2]
```diff
@@ -442,28 +442,28 @@ BEGIN
...
-    ID_REGISTRY_CONNECTNETWORKREGISTRY "Conectare la registrul unui calculator din rețea"
-    ID_REGISTRY_DISCONNECTNETWORKREGISTRY "Deconectare de la registrul calculatorului din rețea"
-    ID_REGISTRY_PRINT "Imprimă toate părțile unui registru"
-/*    ID_HELP_HELPTOPICS "Deschide manualul editorului de registru" */
+    ID_REGISTRY_LOADHIVE "Încarcă un fișier binar în regiștri"
+    ID_REGISTRY_UNLOADHIVE "Eliberează conținutul unui fișier binar din regiștri"
+    ID_REGISTRY_CONNECTNETWORKREGISTRY "Conectare la regiștrii unui calculator din rețea"
+    ID_REGISTRY_DISCONNECTNETWORKREGISTRY "Deconectare de la regiștrii calculatorului din rețea"
+    ID_REGISTRY_PRINT "Imprimă tot sau o parte din regiștri"
+/*    ID_HELP_HELPTOPICS "Deschide termenii de ajutor a editorului de regiștri" */
```
Comment (JoachimHenze) [inline base/applications/regedit/lang/ro-RO.rc:452]: Do not touch this line, you do create a merge issue with an existing PR.

File: base/applications/regedit/lang/ro-RO.rc [hunk 3]
```diff
@@ -83,57 +83,57 @@ BEGIN
...
         MENUITEM "E&xportare…", ID_REGISTRY_EXPORTREGISTRYFILE
         MENUITEM SEPARATOR
-        MENUITEM "Î&ncărcare registru…", ID_REGISTRY_LOADHIVE, GRAYED
-        MENUITEM "Eli&berare registru…", ID_REGISTRY_UNLOADHIVE, GRAYED
+        MENUITEM "Î&ncărcare de regiștri…", ID_REGISTRY_LOADHIVE, GRAYED
+        MENUITEM "&Descărcare de regiștri…", ID_REGISTRY_UNLOADHIVE, GRAYED
         MENUITEM SEPARATOR
         MENUITEM "&Conectare la un registru din rețea…", ID_REGISTRY_CONNECTNETWORKREGISTRY
-        MENUITEM "&Deconectare de la registru din rețea…", ID_REGISTRY_DISCONNECTNETWORKREGISTRY, GRAYED
+        MENUITEM "&Deconectare de la regiștrii din rețea…", ID_REGISTRY_DISCONNECTNETWORKREGISTRY, GRAYED
```
Comment (JoachimHenze) [inline base/applications/regedit/lang/ro-RO.rc:90]: "&Deconectare de la regiștrii din rețea…" collides with MENUITEM "&Descărcare de regiștri…". Change is mandatory.

**Review by JoachimHenze — CHANGES_REQUESTED** [review]
The 3 change-requests are mandatory.

File: base/applications/regedit/lang/ro-RO.rc (hunk 2 above)
Comment (JoachimHenze) [reply base/applications/regedit/lang/ro-RO.rc]: I did resolve the conflicts for you. Please do git pull from that branch before continuing your work on top of it.

File: base/applications/regedit/lang/ro-RO.rc [hunk 4]
```diff
@@ -31,19 +31,19 @@ Opțiuni:\n\
                  Compatibil cu orice alt comutator. Ignorat.\n\
   /R:user.dat    Locația fișierului user.dat urmează să fie modificată.\n\
                  Compatibil cu orice alt comutator. Ignorat.\n\
-  /C             Importă conținutul unui fișier de registru.\n\
-  /D             Șterge o cheie de registru specificată.\n\
-  /E             Exportă conținutul unei chei de registru specificată într-un\n\
+  /C             Importă conținutul unui fișier de regiștri.\n\
```
Comment (JoachimHenze) [inline base/applications/regedit/lang/ro-RO.rc:34]: What is the benefit of all those "registru" -> "regiștri" changes? You do change an already translated term that is often used to one that contains a complex character. Which one of both does romanian MS Windows 2k3 use? Can you prove that with a picture?

Comment (AndreiMiloiu) [reply base/applications/regedit/lang/ro-RO.rc:34]: @JoachimHenze I think you are right. There is no benefit, so I will revert this change. Unfortunately, Windows XP/2k3 is not completely translated in Romanian, so the term registry is not translated anywhere. Hence, I cannot show you any proof. I will revert this change.

File: base/applications/regedit/lang/ro-RO.rc (hunk 1 above)
Comment (AndreiMiloiu) [reply base/applications/regedit/lang/ro-RO.rc:129]: The file is not localized in Romanian, so I cannot send you any proof. I didn't pay attention. I will revert this change.

**Comment by AndreiMiloiu** [comment]
@JoachimHenze I fixed all the issues you found.

**Review by JoachimHenze — APPROVED** [review]
Approved. As a non-native-speaker I cannot spot anymore issues now anymore.

File: base/applications/regedit/lang/ro-RO.rc (hunk 4 above)
Comment (JoachimHenze) [reply base/applications/regedit/lang/ro-RO.rc]: resolved.

File: base/applications/regedit/lang/ro-RO.rc (hunk 1 above)
Comment (JoachimHenze) [reply base/applications/regedit/lang/ro-RO.rc]: resolved

File: base/applications/regedit/lang/ro-RO.rc (hunk 2 above)
Comment (JoachimHenze) [reply base/applications/regedit/lang/ro-RO.rc]: currently resolved.

File: base/applications/regedit/lang/ro-RO.rc (hunk 3 above)
Comment (JoachimHenze) [reply base/applications/regedit/lang/ro-RO.rc]: resolved.

**Pull request merged**